schema_test.go

```diff
@@ -79,6 +79,39 @@ func ExampleBasic() {
 	// /friends/0: {"firstName":"Nas"} "lastName" value is required
 }
 
+func TestTopLevelType(t *testing.T) {
+	schemaObject := []byte(`{
+    "title": "Car",
+    "type": "object",
+    "properties": {
+        "color": {
+            "type": "string"
+        }
+    },
+    "required": ["color"]
+}`)
+	rs := &RootSchema{}
+	if err := json.Unmarshal(schemaObject, rs); err != nil {
+		panic("unmarshal schema: " + err.Error())
```

Comment:
Instead of `panic`, in the context of a test it's better to use `t.Fatal()` or in this case it's it's cousin `t.Fatalf("unmarshal schema: %s", err.Error())`.

Big fan of panic-ie/Fatal calls here tho 😄